### PR TITLE
passport added and conflict resoved

### PR DESCRIPTION
**Pull request by tanay13**

### CODE REVIEW

**Comment by tanay13** [comment]
Screenshot (115)
@YashKumarVerma What these red lines means ? Is it necessary to pass all the travis check to get merged? I am not getting why these tests are failing .

File: src/modules/config/auth.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+   var isAuth=function(req:any,res:any,next:any){
```
Comment (YashKumarVerma) [inline src/modules/config/auth.ts:2]: use const

File: src/modules/config/passport.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const LocalStrategy = require('passport-local').Strategy;
+const bcrypt = require('bcryptjs');
+
+// Load User model
+const User = require('../user/schema.ts');
+
+var auth = function(passport:any) {
```
Comment (YashKumarVerma) [inline src/modules/config/passport.ts:7]: shoud be const

**Comment by tanay13** [comment]
@YashKumarVerma  const added

**Comment by YashKumarVerma** [comment]
> Screenshot (115)
> 
> @YashKumarVerma What these red lines means ? Is it necessary to pass all the travis check to get merged? I am not getting why these tests are failing .

This is from the test coverage report. Not a problem.

**Comment by tanay13** [comment]
@YashKumarVerma Updated travis.yml

**Comment by tanay13** [comment]
@YashKumarVerma Yes i think i did that .. should i do it again ?

**Comment by tanay13** [comment]
@YashKumarVerma  I fixed all the errors now Travis CI build has passed . Now is it fine?

**Pull request merged**